Setup/Patch/Data/SetupModuleDataPatch.php

```diff
@@ -698,10 +583,13 @@ protected function installOrderStatusses(ModuleDataSetupInterface $setup)
 
         return $this;
     }
+
     /**
      * @param ModuleDataSetupInterface $setup
      *
      * @return $this
+     *
+     * @SuppressWarnings(PHPMD.ExcessiveMethodLength)
      */
```

Comment:
this is just a remarkt, not sure if it;s worth fixing at this point. module should not add columns to the order related tables. all module related data should reside in a separate table with a FK to the orders (quote, items, ...) tables. If this module gets installed on a database with 100k+ orders for example the install script will take  a lot of time and might even crash. Doing this properly at this point will require a lot of time and a lot of code changes. not sure if it;s worth changing, but it's sure worth investigating.